docs/faq.md

```diff
@@ -0,0 +1,99 @@
+# FAQ
+
+> Q. How do I download Iris?
+
+A. Please visit [the official Iris website](https://irisshaders.net/)
+
+> Q. Why doesn't the Iris Installer work?
+
+A. You need java, please get it from: [Adoptium.net](https://adoptium.net/?variant=openjdk17&jvmVariant=hotspot)
+
+> Q. How can I configure my shaders?
+
+A. [#663](https://github.com/IrisShaders/Iris/issues/663)
+
+> Q. Why is (insert shader) not working?
+
+A. That shader likely isn't supported currently, but should be in the future.
+
+> Q. How do I disable my shaders?
+
+A. Press K to disable shaders without using the GUI.
+
+> Q. I added Iris, why is my game crashing?
+
+A. There are a few possible reasons:
+
+• You're using an outdated version of Indium, Sodium Extras or Better Sodium Menu.
+
+• You're using an unsupported shader pack.
+
+• You're using macOS, which has limited support.
+
+• You may have an incompatible mod installed.
+
+> Q. What shaders have been tested and are working with Iris?
+
+A. The following shaders have been tested and work well:
+
+• [Complementary Shaders](https://www.curseforge.com/minecraft/customization/complementary-shaders)
+
+• [Prismarine Shaders](https://www.curseforge.com/minecraft/customization/prismarine-shader)
+
+• [BSL Shaders](https://bitslablab.com/bslshaders/)
+
+• [Continuum 1.3](https://continuum.graphics/downloads)
+
+• [Continuum 2.0.4](https://continuum.graphics/downloads)
+
+• [SEUS V11](https://www.sonicether.com/seus/)
+
+• [SEUS Renewed](https://www.sonicether.com/seus/)
```

Comment:
Should we include PTGI here now?

Reply:
Maybe not in the functioning shaderpack section since can have some issues on some hardware and drivers

Reply:
To be fair, Renewed has similar issues and we put it here

Reply:
Ah right